src/components/ResolutionUser.svelte

```diff
@@ -0,0 +1,177 @@
+<script lang="ts">
+  import Tooltip, { Wrapper } from "@smui/tooltip";
+  import { Icon } from "@smui/common";
+  import { Svg } from "@smui/common/elements";
+  import { mdiInformationOutline } from "@mdi/js";
+
+  import { signerAddress } from "../state/eth";
+  import { agent, usersWithEthereumAddress } from "../state/odoo";
+  import type { OdooUserTransformed } from "../types";
+  import Tag from "./Tag.svelte";
+
+  export let ethereumAddress: string;
+  export let title: string = null;
+  export let inline = false;
+  export let hasBg = false;
+  export let hideInfo = false;
+  export let size: "sm" | "md" | "lg" = "md";
+
+  let userDetails: OdooUserTransformed = null;
+
+  $: {
+    if ($usersWithEthereumAddress) {
+      userDetails =
+        $usersWithEthereumAddress[ethereumAddress.toLowerCase()] || null;
+    }
+  }
+</script>
+
+{#if inline}
+  <span class="inline-flex">
+    {#if userDetails}
+      {userDetails.displayName}
+    {:else}
+      <span>{ethereumAddress}</span>
+      {#if !hideInfo && $agent}
+        <Wrapper>
+          <span class="icon-wrapper">
+            <Icon component={Svg} viewBox="0 0 24 24">
+              <path fill="currentColor" d={mdiInformationOutline} />
+            </Icon>
+          </span>
+          <Tooltip yPos="above"
+            >It looks this address is not mapped with odoo data</Tooltip
+          >
+        </Wrapper>
+      {/if}
+    {/if}
+    {#if $signerAddress?.toLowerCase() === ethereumAddress.toLowerCase() && !hideInfo}
+      <Tag label="you" size={size === "sm" ? "xs" : "sm"} />
+    {/if}
+  </span>
+{:else}
+  <div class={`wrapper ${hasBg ? "wrapper--has-bg" : ""} wrapper--${size}`}>
```

Comment:
With svelte you can do `class:wrapper--has-bh={hasBg}` (see [doc](https://svelte.dev/docs#template-syntax-element-directives-class-name)).

Reply:
Ohh nice. Now that I'm thinking I was also used to use https://github.com/JedWatson/classnames (w/ react I've mainly used styled components lately)